bakery/src/tasks/look-up-book.js

```diff
@@ -32,11 +32,11 @@ const task = (taskArgs) => {
           cp output-producer/version book/version
           cp output-producer/collection_style book/style
           cp output-producer/content_server book/server
-          pdf_filename="$(cat book/collection_id)-$(cat book/version)-$(cat book/server_name)-$(cat book/job_id).pdf"
           echo "$pdf_filename" > book/pdf_filename
           wget -q -O jq 'https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64' && chmod +x jq
-          server_name="$(cat output-producer/job.json | ./jq -r '.content_server.name')"
-          echo "$server_name" >book/server_name
+          server_shortname="$(cat output-producer/job.json | ./jq -r '.content_server.name')"
+          echo "$server_shortname" >book/server_shortname
+          pdf_filename="$(cat book/collection_id)-$(cat book/version)-$(cat book/server_shortname)-$(cat book/job_id).pdf"
```

Comment:
Should the line `echo "$pdf_filename" > book/pdf_filename` also be moved after this to write it out?

Reply:
Good catch

Reply:
makes me think that perhaps it's best to build that filename via a function with arguments defined? seems like an easy thing to mess up so why not have it be generated?

Reply:
Hmmm...I'm not sure I understand. It's computed in one place here and saved for future reference. So, in effect, this is a function execution for other tasks. @m1yag1 is your suggestion to add a function def in the task bash or something?